### DISCUSSION TITLE
Bad performance on decent pc

### DISCUSSION THREAD

**Discussion by tonaz** [praydog/REFramework#320] [General Questions]
Hello! i get bad performance on both RE2 and RE7, always AWS kicking in, some stuttering, zero "headroom".
At very low settings as well.
Ryzen 7 3800x, RTX 2080ti, 32GB ram.
Is there anything i am doing wrong/missing?
Thanks!

**Comment by praydog** [comment]
Read the troubleshooting wiki. https://cursey.github.io/reframework-book/troubleshooting/VR-Troubleshooting.html
